concepts/05 Widgets/Scheduler/060 Views/020 Customize Individual Views.md

```diff
@@ -98,6 +98,114 @@ The following code defines three views: the first is not customized, the second
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler
+            :data-source="schedulerData"
+            :current-date="currentDate"
+            :views="views"
+            :resources="resources"
+            time-cell-template="TimeCellTemplateSlot">
+
+            <template #TimeCellTemplateSlot="appointment">
+                <i style="color: green">{{appointment.text}}</i>
+            </template>
+        />
+    </template>
+
+    <script>
+    import DxScheduler from 'devextreme-vue/scheduler';
+
+    export default {
+        components: {
+            DxScheduler
+        },
+        data() {
+            return {
+                schedulerData: [{
+                    text: "Google AdWords Strategy",
+                    ownerId: [2],
+                    startDate: new Date(2016, 1, 1, 9, 0),
+                    endDate: new Date(2016, 1, 1, 10, 30)
+                }, {
+                    text: "New Brochures",
+                    ownerId: [1],
+                    startDate: new Date(2016, 1, 1, 11, 30),
+                    endDate: new Date(2016, 1, 1, 14, 15)
+                },
+                // ...
+                ],
+                currentDate: new Date(2016, 1, 1),
+                views: [
+                    "month",
+                    { type: "day", cellDuration: 60, timeCellTemplate: 'timeCellTemplate' },
+                    { type: "workWeek", groups: ["ownerId"] }
+                ],
+                resources: [{
+                    fieldExpr: "ownerId",
+                    dataSource: [
+                        { text: "Samantha Bright", id: 1, color: "#cb6bb2" },
+                        { text: "John Heart", id: 2, color: "#56ca85" }
+                    ]
+                }]
+            }
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import Scheduler from 'devextreme-react/scheduler';
+
+    const schedulerData: [{
```

Comment:
An equal sign `=` should be used to initialize a constant. Please fix this error in all your PRs.